google/cloud/logging_v2/logger.py

```diff
@@ -155,7 +157,13 @@ def _do_log(self, client, _entry_class, payload=None, **kw):
             entry = _entry_class(**kw)
 
         api_repr = entry.to_api_repr()
-        client.logging_api.write_entries([api_repr])
+        entries = [api_repr]
+        if google.cloud.logging_v2.instrumentation_emitted is False:
+            partial_success = True
+            entries = add_instrumentation(entries, **kw)
```

Comment:
Is there any harm in adding `partial_success=True` to a call to `write_entries` with a single entry?  Because that's the current behavior for log entries if the first log entry already had instrumentation info.

Reply:
Hm... When you add a record with instrumentation data, it means there are at least 2 records.